how-to-docs/drone-how-to.md

```diff
@@ -315,70 +315,74 @@ Drone will automatically execute that step when a new pull request is raised.
 
-```bash
-[INFO] 2016/10/20 10:17:40 main.go:158: deploying deployment/induction-hello-world
-[ERROR] 2016/10/20 10:17:40 main.go:160: error: You must be logged in to the server (the server has asked for the client to provide credentials)
-[ERROR] 2016/10/20 10:17:40 main.go:130: exit status 1
-```
+You will need to add `KUBE_TOKEN` and `KUBE_SERVER` as drone secrets.
 
-This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to DSP.
+You can add and view secrets through the Drone UI. Go to Drone and select your repo, then click the icon in the top right and select **Secrets**. You should be presented with a list of the secrets for that repo (if there are any) and you should be able to add secrets giving them a name and value. Add the `KUBE_TOKEN` and `KUBE_SERVER` secrets with their respective values.
 
-The kube `deploy.sh` scripts relies on 3 environment variables:
 
-- `KUBE_NAMESPACE` - the kubernetes namespace you wish to deploy to. **You need to provide the kubernetes namespace as part of the deployment job**.
+Alternatively, you can use this command to add the `KUBE_TOKEN` secret:
 
-- `KUBE_TOKEN` - this is the token used to authenticate against the kubernetes cluster. **You need to add this as a secret to your build step. You can [request a kubernetes token here](https://github.com/UKHomeOffice/application-container-platform-bau/issues/new)**. In this particular case, the secret was added with:
+```bash
+$ drone secret add --image=quay.io/ukhomeofficedigital/kd:v0.2.3 --conceal ukhomeoffice/<your_repo> KUBE_TOKEN <your_token>
+```
 
-  ```bash
-  $ drone secret add --image=quay.io/ukhomeofficedigital/kd:v0.2.3 --conceal  UKHomeOffice/<your_repo> KUBE_TOKEN <your_token>
-  ```
+Adding the `KUBE_SERVER` will be similar.
 
-- `KUBE_SERVER` - this is the address of the kubernetes cluster. There are four environment variables set as an organisational secret in Drone: `KUBE_SERVER_DEV`,  `KUBE_SERVER_OPS`, `KUBE_SERVER_PROD` and `KUBE_SERVER_CI`. You can verify that the secrets are present with:
+You can verify that the secrets for your repo are present with:
+
+```
+$ drone secret ls ukhomeoffice/<your-repo>
+```
 
-  ```
-  $ drone org secret ls UKHomeOffice
-  ```
+_Please note that you need to be an admin to issue this command._
 
-  _Please note that you need to be an admin to issue this command._
 
-You need to reassign one of those four variables to `KUBE_SERVER` before your script runs, [like in this case](https://github.com/UKHomeOffice/kube-node-hello-world/blob/99af304ce0b894e8f0db1c05780cf6512741516d/deploy.sh#L4).
+Once the secrets have been added, execute the deployment script by adding a new step to your drone pipeline:
```

Comment:
Adding this pipeline step doesn't quite "execute the deployment script", so this could be worded better.

Also – and this may be covered elsewhere, but should probably be mentioned here - I believe once you have this pipeline step added and drone has picked it up, then you need to use `drone-trigger` to actually trigger the build.

Reply:
Earlier on in the doc (the Deployments and promotions section) it mentions using `drone deploy` to deploy a build. Does that not work any more or is `drone-trigger` preferred?

I do agree that this should be worded better though.

Reply:
That's the first I'm hearing of `drone deploy`! That may well be a replacement for `drone-trigger` in our new Drone instance. @vaijab may be able to tell us definitively.

Reply:
It might be the other way around as `drone deploy` is in the old docs but `drone-trigger` isn't.

Reply:
my understanding is that drone-trigger runs drone deploy but from within drone, for automatic deployments, and drone deploy would be used for manual deployments